amoro-ams/src/main/java/org/apache/amoro/server/AmoroManagementConf.java

```diff
@@ -380,6 +380,12 @@ public class AmoroManagementConf {
           .defaultValue(Duration.ofSeconds(3))
           .withDescription("Optimizer polling task timeout.");
 
+  public static final ConfigOption<Duration> OPTIMIZING_REFRESH_GROUP_INTERVAL =
+      ConfigOptions.key("self-optimizing.refresh-group-interval")
+          .durationType()
+          .defaultValue(Duration.ofSeconds(30))
+          .withDescription("Optimizer group refresh interval.");
+
```

Comment:
Do we need to add this configuration to the Helm chart and configuration document (in the next PR)?